docs/samples/deployments/redacting-telemetry/docker-compose.yml

```
@@ -0,0 +1,99 @@
+  loki:
+    image: grafana/loki:2.9.3
+    ports:
+      - "3100:3100"
+    command: -config.file=/etc/loki/local-config.yaml
+    networks:
+      - telemetry
+
+  # Tempo - distributed tracing
+  tempo:
+    image: grafana/tempo:2.3.1
+    command: [ "-config.file=/etc/tempo.yaml" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)